cellengine/resources/plot.py

```diff
@@ -18,72 +18,71 @@ def __attrs_post_init__(self):
     @classmethod
     def get(
         cls,
-        experiment_id,
+        experiment_id: str,
         fcs_file_id: str,
         plot_type: str,
         x_channel: str,
         y_channel: str,
         z_channel: str = None,
         population_id: str = None,
-        as_dict=False,
+        as_dict: bool = False,
         **kwargs,
     ) -> Plot:
         """
         Args:
-            experiment_id (str): ID of the experiment to which the file belongs.
-            fcs_file_id (str): ID of file for which to build a plot.
-            plot_type (str): "contour", "dot", "density" or
+            experiment_id: ID of the experiment to which the file belongs.
+            fcs_file_id: ID of file for which to build a plot.
+            plot_type: "contour", "dot", "density" or
```

Comment:
Q: in some places you're adding type annotations in these doc strings, in other places removing. Are they redundant if they're in the function argument typings?

Reply:
Correct, they are redundant if they are in fn arg typings. They are thus required when we specify kwarg typings